pulp_rpm/app/models/repository.py

```diff
@@ -102,6 +108,74 @@ class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
 
 
+class UlnRemote(Remote):
+    """
+    Remote for "uln" content.
+    """
+
+    TYPE = "uln"
+
+    uln_user = models.CharField(max_length=512, null=True)
+
+    uln_pwd = models.CharField(max_length=512, null=True)
+
+    uln_server_base_url = models.CharField(max_length=512, null=True)
```

Comment:
Since we now have a dedicated ULN remote, we can simply make these fields required, and get rid of the exceptions handling for when they are not set. Users simply have to provide these fields when creating an ULN remote.

Reply:
Done :)

Reply:
By this I mean we can set the corresponding serializer fields to `required=True` and then completely get rid of `NoUlnCredentialsError` since that is then an exception that can never happen.